api/lib/reducers/cosmosV0-reducers.js

```diff
@@ -162,7 +162,8 @@ function networkAccountReducer(address, validators) {
   return {
     name: validator ? validator.name : undefined,
     address: address || '',
-    picture: validator ? validator.picture : ''
+    picture: validator ? validator.picture : '',
+    validator
```

Comment:
adding the whole validator object as well?

Reply:
yes. gql will filter out